spec/helpers.rb

```diff
@@ -2,6 +2,15 @@ def auth_user_headers
   user.create_new_auth_token
 end
 
+def another_auth_user_headers
+  another_user.create_new_auth_token
+end
+
 def parsed_data
   JSON.parse(response.body)
 end
+
+def conversation_user
```

Comment:
Why this helper? 🤔

Reply:
To refer to the conversation_user instance im currrently using. 
This is used in conversation spec to check the unread_messages on message related methods.

Reply:
I don't think it's necessary to add this.